concordium-contracts-common/src/schema_json.rs

```diff
@@ -930,6 +930,41 @@ fn write_bytes_from_json_schema_type<W: Write>(
     }
 }
 
+impl Fields {
+    pub fn field_to_json(&self) -> serde_json::Value {
+        match self {
+            Fields::Named(vec) => {
+                let mut map = Map::new();
+
+                for (name, field) in vec.iter() {
+                    map.insert(name.to_string(), field.schema_type_to_json());
+                }
+
+                serde_json::Value::Object(map)
+            }
+            Fields::Unnamed(vec) => {
+                let mut new_vector: Vec<Value> = vec![];
```

Comment:
```suggestion
                let mut new_vector = Vec::new();
```
I think it is slightly more idiomatic to use `Vec::new()` instead of the macro for empty vectors. And can't the compiler infer the type?